angular-client/src/pages/charging-page/charging-page.component.html

```diff
@@ -3,7 +3,7 @@
 
   <charging-page-mobile style="width: 100%" *ngIf="isMobile; else isDesktop" [time]="time" />
   <ng-template #isDesktop>
-    <vstack style="width: 100%" spacing="15px">
+    <vstack style="width: 100%; min-width: 1290px" spacing="15px">
```

Comment:
doing minWidth at this top level is still fairly janky. You should only be using this on components that will look fucked if they squished too much, everything else should shrink/ stretch to fill the rest of the space

Reply:
will do 🫡.